src/esy/buildEjectCommand/index.js

```diff
@@ -15,6 +15,8 @@ const outdent = require('outdent');
 const {flattenArray} = require('../Utility');
 
 const RUNTIME = fs.readFileSync(require.resolve('./runtime.sh'), 'utf8');
+const fastReplaceString = fs.readFileSync(require.resolve('../../../node_modules/.bin/fastreplacestring'));
```

Comment:
Does it mean that we are going to put a compiled executable into an ejected build? I think we need to put sources there and a make rule to compile it — ejected build might be copied to another machine with another architecture on which an already compiled executable won't work.